browser-test/eslint.config.mjs

```diff
@@ -22,7 +22,6 @@ export default defineConfig([{
         "typescriptEslint/recommended-type-checked",
         google,
         prettier,
-        "playwright/recommended",
```

Comment:
This goes away because we have custom rules defined and having it here and playwright defined above fails.